CHANGELOG.md

```diff
@@ -32,6 +32,10 @@ antennas with data.
 - A bug in reading UVH5 files with antenna names saved as variable length strings
 that was introduced in v3.0.0.
 
+### Changed
+- Made it possible to *not* return the `interp_basis_vector` array from beam
+interpolators.
+
```

Comment:
This needs to be moved up to be in the Unreleased section.

Reply:
Ah yeah, things were released in between ;-)